### PR TITLE
acl_test improvements unit test 

### PR DESCRIPTION
**Pull request by eleemhui**

### CODE REVIEW

**Review by leonmatt — APPROVED** [review]
LGTM

File: test/testcne/acl_test.c [hunk 1]
```diff
@@ -1186,16 +1194,16 @@ test_invalid_layout(void)
...
 
     for (i = 0; i < (int)CNE_DIM(results); i++) {
         if (results[i] != invalid_layout_data[i].allow) {
-            cne_printf("Line %i: Wrong results at %i "
-                       "(result=%u, should be %u)!\n",
-                       __LINE__, i, results[i], invalid_layout_data[i].allow);
+            tst_error("Line %i: Wrong results at %i "
+                      "(result=%u, should be %u)!",
+                      __LINE__, i, results[i], invalid_layout_data[i].allow);
             goto err;
```
Comment (sweeksBigBlue) [inline test/testcne/acl_test.c:1207]: same here: tst_error("test_invalid_layout() failed") in the err: goto

**Review by sweeksBigBlue — CHANGES_REQUESTED** [review]
It would also be nice to add tst_ok("PASS --- <some_message>") above test_end() in the happy path and tst_error("FAILED --- <some_message>") in the err: case. These would be added to the acl_main on lines 1793 and 1797 respectively.

File: test/testcne/acl_test.c [hunk 2]
```diff
@@ -324,9 +324,9 @@ test_classify_run(struct cne_acl_ctx *acx, struct ipv4_7tuple test_data[], size_
...
         result = results[i * CNE_ACL_MAX_CATEGORIES + ACL_DENY];
         if (result != test_data[i].deny) {
-            cne_printf("Line %i: Error in deny results at %i "
-                       "(expected %" PRIu32 " got %" PRIu32 ")!\n",
-                       __LINE__, i, test_data[i].deny, result);
+            tst_error("Line %i: Error in deny results at %i "
+                      "(expected %" PRIu32 " got %" PRIu32 ")!",
+                      __LINE__, i, test_data[i].deny, result);
             ret = -EINVAL;
             goto err;
```
[the remark this answers is not in the document]
Comment (eleemhui) [reply test/testcne/acl_test.c:331]: done

File: test/testcne/acl_test.c (hunk 1 above)
Comment (eleemhui) [reply test/testcne/acl_test.c:1210]: done

**Review by sweeksBigBlue — APPROVED** [review]

**Review by byron-marohn — APPROVED** [review]

**Comment by byron-marohn** [comment]
pulled into test_improvements branch to submit upstream

**Pull request closed without merging**